neutrino-custom/not-test.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+const Md5HashPlugin = require('webpack-md5-hash');
+
+module.exports = neutrino => neutrino.config
+    // Include files from node_modules in the separate, more-cacheable vendor chunk:
+    .entry('vendor')
+        .merge([
+            'angular', 'angular-cookies', 'angular-local-storage', 'angular-resource',
+            'angular-route', 'angular-sanitize', 'angular-toarrayfilter', 'angular-ui-bootstrap',
+            'angular-ui-router', 'bootstrap/dist/js/bootstrap', 'hawk', 'jquery', 'jquery.scrollto',
+            'js-yaml', 'mousetrap', 'react', 'react-dom', 'taskcluster-client'
+        ])
+        .end()
+    // Ensure chunkhashes for unchanged chunks don't change -- this allows the bundled vendor
+    // file to keep its hash and stay cached in the user's browser when it hasn't been updated.
+    .output
+        .chunkFilename('[chunkhash].[id].chunk.js').end()
+    .plugin('hash')
```

Comment:
Also, I know this is pre-existing, but I don't think the hashing should be done in development, according to https://webpack.js.org/guides/caching/ 

> Don’t use [chunkhash] in development since this will increase compilation time. Separate development and production configs and use [name].js for development and [name].[chunkhash].js in production. 

ie: not-test.js probably shouldn't exist, and all of this be moved into production.js perhaps? (Though perhaps the vendor splitting should be in the common config?)